profile.md

```diff
@@ -324,7 +324,22 @@ The contents of the claim may either be a string or URL; we do not currently pro
   </tr>
 </table>
 
+The `aud` claim provides an explicit statement of the intended audience for the token; it is an important mechanism for restricting the scope of the token to one or more relying parties.  Note requiring `aud` is atypical: we believe it is better to have the issuer explicitly state the token may be used by any relying party as opposed to assuming its absence indicates no restrictions.
 
+Relying parties are recommended to accept audiences based on the endpoint name most commonly associated with the provided service.
+That is, if a client would want to access the resource `https://wlcg.example.com/base/file.root`, then the recommended audience for the relying party at this endpoint would be:
+
+```
+https://wlcg.example.com
+```
+
+The audience SHOULD be normalized according to Section 6 of RFC 3986; that is, trailing slashes are discouraged and default ports should not be included.  However, the comparison against the value MUST be done as a case sensitive string as specified in Section 4.1.3 RFC 7519.
+
+Audiences of this form are preferred (as opposed to a human-readable "site name" such as `WLCG_Site_Foo`) because the user can derive the correct audience from a given URL then determine the correct token to request.
```

Comment:
`the user can derive the correct audience from a given URL`

I'm not sure that this necessarily follows.  A single service may have multiple endpoints, with different port numbers and host names.  The service *could* all accept tokens with the same audience, since it's a single service.  However, this scheme would force the client to have a separate token for each endpoint.

Reply:
That's not what is intended by the text ... in fact, it says that every server behind the load balancer should accept the audience from the server itself.

Just because the user has the _ability_ to derive the audience from the URL they are given does not mean it _must_ use that audience for every internal redirect.